plugins/maven/src/main/java/org/jetbrains/idea/maven/execution/MavenExternalParameters.java

```diff
@@ -527,6 +527,11 @@ private static void encodeCoreAndRunnerSettings(MavenGeneralSettings coreSetting
     if (coreSettings.getOutputLevel() == MavenExecutionOptions.LoggingLevel.DEBUG) {
       cmdList.add("--debug");
```

Comment:
Have added a new level for error output (ERROR). The four logging levels are:

1. All (INFO, default)
2. Warnings and errors (WARN, -q, --quiet) 
3. Debug output (DEBUG, -X, --debug)
4. Error output (ERROR, -e, --errors)